YAXLib/Pooling/PoolSettings.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (C) Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors.
+// Licensed under the MIT license.
+
+#nullable enable
+
+namespace YAXLib.Pooling
+{
+    internal static class PoolSettings
+    {
+        public static bool CheckReturnedObjectsExistInPool { get; set; } = false;
+        public static bool IsPoolingEnabled { get; set; } = true;
```

Comment:
Just useful for comparing performance. Remove?

Reply:
Vote for removal.

Reply:
Agree for `IsPoolingEnabled`. For `CheckReturnedObjectsExistInPool ` see [below](https://github.com/YAXLib/YAXLib/pull/199#discussion_r926096902)